fastlane/spec/commands_generator_spec.rb

```diff
@@ -5,6 +5,8 @@
     ENV['DOTENV'] = nil
     fastlane_folder = File.absolute_path('./fastlane/spec/fixtures/dotenvs/withFastfiles/parentonly/fastlane')
     allow(FastlaneCore::FastlaneFolder).to receive(:path).and_return(fastlane_folder)
+    allow(FastlaneCore::Env).to receive(:truthy?).and_return(:default)
+    allow(FastlaneCore::Env).to receive(:truthy?).with('FASTLANE_SKIP_DOCS').and_return(true)
```

Comment:
@KrauseFx Uhh, no after all. I tried to define these in spec_helper, and it had other problems to cause test failure.